### PR TITLE
Update exposed endpoints on kraken

### PR DESCRIPTION
**Pull request by jrozsa**
Moved closer to desired 1.5.
All endpoints except api/uniflow/workflow/{workflowId}/remove should be there. Unfortunately it is in conflict with
the other endpoints in desired state. 
My proposal for fix: api/uniflow/workflow/remove/{workflowId}

### CODE REVIEW

File: config/krakend/settings/workflow_proxy_settings_deprecated.json [hunk 1]
```diff
@@ -46,9 +41,9 @@
             "url_pattern": "/executions"
         },
         {
-            "endpoint": "/api/uniflow/workflow/{a}",
-            "method": "DELETE",
-            "url_pattern": "/workflow/{a}"
```
Comment (Jozefiel) [inline config/krakend/settings/workflow_proxy_settings_deprecated.json:51]: delete_exec

File: config/krakend/settings/workflow_proxy_settings_deprecated.json [hunk 2]
```diff
@@ -5,11 +5,6 @@
             "method": "PUT",
             "url_pattern": "/metadata"
         },
-        {
-            "endpoint": "/api/uniflow/metadata/workflow/{name}/{version}",
-            "method": "GET",
-            "url_pattern": "/metadata/workflow/{name}/{version}"
```
Comment (Jozefiel) [inline config/krakend/settings/workflow_proxy_settings_deprecated.json:11]: this endpoint is included in the Public API proposal document. why was removed?

File: config/krakend/settings/workflow_proxy_settings.json [hunk 3]
```diff
@@ -131,9 +126,9 @@
             "url_pattern": "/editableworkflows"
         },
         {
-            "endpoint": "/api/uniflow/workflow/{a}/{b}",
```
Comment (Jozefiel) [inline config/krakend/settings/workflow_proxy_settings.json:134]: this also cover desired AG api /api/uniflow/workflow/{workflowId}/remove

File: config/krakend/settings/workflow_proxy_settings.json [hunk 4]
```diff
@@ -46,14 +46,14 @@
...
         },
         {
-            "endpoint": "/api/uniflow/metadata/workflow/{a}/{b}",
+            "endpoint": "/api/uniflow/metadata/workflow/{name}/{version}",
             "method": "DELETE",
-            "url_pattern": "/metadata/workflow/{a}/{b}"
+            "url_pattern": "/metadata/workflow/{name}/{version}"
         },
         {
-            "endpoint": "/api/uniflow/workflow/{a}",
```
Comment (Jozefiel) [inline config/krakend/settings/workflow_proxy_settings.json:54]: this endpoint also cover /api/uniflow/workflow/:workflowId?includeTask={true|false}, which is included in public API proposal. is that ok?

File: config/krakend/settings/workflow_proxy_settings_deprecated.json (hunk 2 above)
Comment (jrozsa) [reply config/krakend/settings/workflow_proxy_settings_deprecated.json:11]: Moved to not deprecated file

Comment (Jozefiel) [reply config/krakend/settings/workflow_proxy_settings_deprecated.json:11]: But there is a different method used.

**Comment by jrozsa** [comment]
Update:

        {
            "endpoint": "/api/uniflow/workflow/{a}/remove",
            "method": "DELETE",
            "url_pattern": "/workflow/{a}/remove"
        },
 conflicts with
       {
            "endpoint": "/api/uniflow/workflow/bulk/terminate",
            "method": "DELETE",
            "url_pattern": "/workflow/bulk/terminate"
        },
(both in desired 1.5)
=> solution:
        {
            "endpoint": "/api/uniflow/workflow/{a}/{b}",
            "method": "DELETE",
            "url_pattern": "/workflow/{a}/{b}"
        }

==========================================================
        {
            "endpoint": "/api/uniflow/workflow/{a}",
            "method": "GET",
            "url_pattern": "/workflow/{a}"
        },
conflicts with 
        {
            "endpoint": "/api/uniflow/workflow/search",
            "method": "GET",
            "url_pattern": "/workflow/search"
        }
(both in desired 1.5)

=> solution:
        {
            "endpoint": "/api/uniflow/workflow/{a}",
            "method": "GET",
            "url_pattern": "/workflow/{a}"
        }

**Review by Jozefiel — APPROVED** [review]

**Pull request merged**